app/views/api/stripe_transaction_refunds/_stripe_transaction_refund.json.jbuilder

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
```

Comment:
I don't believe this file has anything to do with the specific change.

Reply:
Done!